src/bindings/python/flux/core/handle.py

```diff
@@ -173,3 +178,44 @@ def get_rank(self):
 
     def attr_get(self, attr_name):
         return self.flux_attr_get(attr_name).decode("utf-8")
+
+    def reactor_run(self, reactor=None, flags=0):
+        """
+        Run reactor associated with this Flux handle or reactor argument
+        if it is provided. Sets a signal watcher for SIGINT to return
+        from the reactor on Ctrl-C, and raise KeyboardInterrupt.
+        """
+        import signal
```

Comment:
Since `signal` is part of the python stdlib, it should be fine (famous last words) to stick this at the top of files, which is the generally preferred practice in python.  I believe the `from flux.message import *Watcher` calls are within the functions to avoid circular references on initial import.

Reply:
Ok great! I fall into bad habits from other languages I fear!